From f8295fff932cee091e7d52bf6bda454c91209e4e Mon Sep 17 00:00:00 2001
From: Kevin Cui <krazycavin@gmail.com>
Date: Tue, 30 Apr 2019 22:22:06 +0200
Subject: [PATCH] Unset bold

---
 lib/section.zsh | 6 +++---
 1 file changed, 3 insertions(+), 3 deletions(-)

diff --git a/lib/section.zsh b/lib/section.zsh
index 4efb8e2..8a11972 100644
--- a/lib/section.zsh
+++ b/lib/section.zsh
@@ -20,9 +20,9 @@ spaceship::section() {
   spaceship_prompt_opened=true
   echo -n "%{%b%}" # unset bold
 
-  echo -n "%{%B$color%}" # set color
-  echo -n "$content"     # section content
-  echo -n "%{%b%f%}"     # unset color
+  echo -n "%{$color%}" # set color
+  echo -n "$content"   # section content
+  echo -n "%{%f%}"     # unset color
 
   echo -n "%{%B%}" # reset bold, if it was diabled before
   if [[ $SPACESHIP_PROMPT_SUFFIXES_SHOW == true ]]; then
-- 
2.21.0

