From ebef774eda2dfdccbbff84e48bd8cd632b72ba69 Mon Sep 17 00:00:00 2001
From: Kevin Cui <krazycavin@gmail.com>
Date: Sun, 24 Mar 2019 12:51:41 +0100
Subject: [PATCH] Sort dirs and files by modified time

---
 fff | 7 +++++--
 1 file changed, 5 insertions(+), 2 deletions(-)

diff --git a/fff b/fff
index 2b9775e..e217a08 100755
--- a/fff
+++ b/fff
@@ -195,8 +195,10 @@ read_dir() {
     # If '$PWD' is '/', unset it to avoid '//'.
     [[ $PWD == / ]] && PWD=
 
-    for item in "$PWD"/*; do
-        if [[ -d $item ]]; then
+    OIFS="$IFS"
+    IFS=$'\n'
+    for item in $(find "$PWD"/* -prune -exec 'stat' -c '%Y|%n' {} \; | sort -t '|' -k 1 -nr | cut -d '|' -f 2); do
+        if [[ -d "$item" ]]; then
             dirs+=("$item")
             ((item_index++))
 
@@ -208,6 +210,7 @@ read_dir() {
             files+=("$item")
         fi
     done
+    IFS="$OIFS"
 
     list=("${dirs[@]}" "${files[@]}")
 
-- 
2.21.0

