--- z.lua.orig	2023-06-05 18:30:13.805800981 +0200
+++ z.lua	2023-06-05 18:30:11.009134462 +0200
@@ -24,6 +24,9 @@
 --     * z -I foo     # cd with interactive selection using fzf
 --     * z -b foo     # cd to the parent directory starting with foo
 --     * z -b foo bar # replace foo with bar in cwd and cd there
+--     * z            # cd ..
+--     * z -          # cd -
+--     * z <num>      # cd .. with <num> times
 --
 -- Bash Install:
 --     * put something like this in your .bashrc:
@@ -2137,6 +2140,26 @@
 	local arg_subdir=""
 	local arg_inter=""
 	local arg_strip=""
+
+    if [ -z ${1:-} ]; then
+        cd ..
+        return
+    fi
+
+    if expr "$1" : '^[0-9]\+$' > /dev/null; then
+        i=0
+        while [ "$i" -lt "$1" ];do
+            cd ..
+            i=$((i+1))
+        done
+        return
+    fi
+
+    if [ "$1" = "-" ]; then
+        cd -
+        return
+    fi
+
 	if [ "$1" = "--add" ]; then
 		shift
 		_ZL_RANDOM="$RANDOM" "$ZLUA_LUAEXE" "$ZLUA_SCRIPT" --add "$@"
