--- monotone.vim	2020-09-21 15:47:10.023860913 +0200
+++ monotone_mod.vim	2020-09-21 15:46:25.303859022 +0200
@@ -129,8 +129,8 @@
 
 	" UI/special
 	call s:Hi('ColorColumn', 'NONE', s:color_dark_2, 'NONE', 234, 'NONE')
-	call s:Hi('CursorLine', 'NONE', s:color_dark_1, 'NONE', 234, 'NONE')
-	call s:Hi('CursorLineNr', s:color_bright_2, s:color_dark_1, 'NONE', 235, 'NONE')
+	call s:Hi('CursorLine', 'NONE', s:color_dark_2, 'NONE', 234, 'NONE')
+	call s:Hi('CursorLineNr', s:color_bright_2, s:color_dark_2, 'NONE', 235, 'NONE')
 	call s:Hi('Folded', s:color_normal, s:color_dark_1, 252, 235, 'italic')
 	call s:Hi('Search', s:color_dark_3, s:color_hl_2, 16, 214, 'bold')
 	call s:Hi('IncSearch', s:color_dark_3, s:color_hl_2, 16, 214, 'bold,reverse')
@@ -149,7 +149,8 @@
 	call s:Hi('MoreMsg', s:color_hl_3, 'NONE', 153, 'NONE', 'bold')
 
 	" Parens
-	call s:Hi('MatchParen', s:color_dark_3, s:color_hl_2, 16, 214, 'NONE')
+	call s:Hi('MatchParen', s:color_dark_3, s:color_bright_1, 16, 214, 'NONE')
+	call s:Hi('MatchParenCur', s:color_dark_3, s:color_hl_2, 16, 214, 'NONE')
 	hi link ParenMatch MatchParen
 
 	" Popup menu
