diff --git a/colors/monotone.vim b/colors/monotone.vim
index ab960ec..5e69cd3 100644
--- a/colors/monotone.vim
+++ b/colors/monotone.vim
@@ -148,7 +148,7 @@ function s:MonotoneColors(color, secondary_hue_offset, emphasize_comments, empha
 	call s:Hi('MoreMsg', s:color_hl_3, 'NONE', 153, 'NONE', 'bold')
 
 	" Parens
-	call s:Hi('MatchParen', s:color_dark_3, s:color_hl_2, 16, 214, 'NONE')
+	call s:Hi('MatchParen', s:color_dark_3, s:color_bright_1, 16, 214, 'NONE')
 	hi link ParenMatch MatchParen
 
 	" Popup menu
